src/meta/model_v2/src/table.rs

```diff
@@ -123,6 +122,7 @@ pub struct Model {
     pub cleaned_by_watermark: bool,
     pub description: Option<String>,
     pub version: Option<TableVersion>,
+    pub retention_seconds: u32,
```

Comment:
Can only be defined as `i32`, and since it's not `Option` you need to define `retention_seconds` as not null in migration.

Reply:
This reminds me of whether it's possible to integrate the test for meta store migration in CI, or if we rely on compatibility testing for such changes. cc @kwannoel

Reply:
I think I can change it to `Option<u32> `

Reply:
fixed

Reply:
>  or if we rely on compatibility testing for such changes.

For backwards compat testing, we run some e2e tests, re-create the cluster, and check everything works as expected. I suppose it can catch most cases.

I'm not sure if there's some corner cases that would be missed.